tox.ini

```diff
@@ -25,6 +25,11 @@ deps =
 setenv =
   DJANGO_SETTINGS_MODULE=config.settings.test
   PYTHONPATH={toxinidir}/cloudigrade
+passenv =
+  AWS_SECRET_ACCESS_ID
```

Comment:
How come we need real values for tox?

Reply:
I have this because the houndigrade script still needs to be able to talk to SQS to send its results, and passing these through individually is easier than trying to construct and pass AWS_SQS_URL since you'd have to url-escape the key and secret in order to build it. I actually started down that road before, and I gave up when the escapes got ugly.

I'm happy to entertain alternate solutions if you have one. 🙂

Reply:
Good point. I've replaced these with bogus values in the latest commit. We need some values here because of how we construct the SQS URL in the settings, and we can't just set defaults in `test.py` because they need to be present when `base.py` loads which happens _before_ `test.py`.